python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -221,6 +221,8 @@ def setDefaults(self):
             "STREAK", "INJECTED", "INJECTED_TEMPLATE"]
         self.skySources.avoidMask = ["DETECTED", "DETECTED_NEGATIVE", "BAD", "NO_DATA", "EDGE"]
 
+        self.maskStreaks.onlyMaskDetected = False
```

Comment:
This is a great place to set it, thank you - suggest one final inline comment e.g. `# Set the streak mask along the entire fit line, not only where the detected mask is set`